inference-engine/src/mkldnn_plugin/utils/debug_capabilities.h

```diff
@@ -4,7 +4,44 @@
 #pragma once
 
 #ifdef CPU_DEBUG_CAPS
-#   define ENABLE_CPU_DEBUG_CAP(_x) _x;
-#else
-#   define ENABLE_CPU_DEBUG_CAP(_x)
-#endif
+
+#include <map>
+#include <string>
+#include <vector>
+
+#define ENABLE_CPU_DEBUG_CAP(_x) _x;
+
+namespace MKLDNNPlugin {
+namespace DebugCaps {
```

Comment:
Maybe DebugUtils?

Reply:
Sure I can update this, but just wondering why would we use Utils vs Capabilities here?